### ISSUE TITLE
Conflicts with `bevy_console`?

### ISSUE THREAD

**Issue by darthdeus** [Nilirad/bevy_prototype_lyon#126]
When using `bevy_prototype_lyon` together with [`bevy_console`](https://github.com/RichoDemus/bevy-console) it seems to crash with

```
thread 'main' panicked at 'Stage already exists: UiFrameEnd.', /home/darth/.cargo/registry/src/github.com-1ecc6299db9ec823/bevy_ecs-0.5.0/src/schedule/mod.rs:137:13
```

Removing `bevy_console` from the project fixes the issue. Not sure if the problem is on the side of `bevy_console` or `bevy_prototype_lyon` or both?

**Comment by nejinx** [comment]
This also happens if you use bevy_prototype_lyon and the heron crate. Tested with heron version 3 with the "debug-2d" feature on.

> thread 'main' panicked at 'Stage already exists: Shape.', C:\Users\...\.cargo\registry\src\github.com-1ecc6299db9ec823\bevy_ecs-0.7.0\src\schedule\mod.rs:150:9

Cargo.toml line for heron:
`heron = { version = "3", features = ["2d", "debug-2d"] }`

once you remove debug-2d this issue disappears.

**Comment by tmills9208** [comment]
I was also having this issue with using [bevy_retrograde](https://github.com/katharostech/bevy_retrograde/), which uses heron as well. This gives better insight on the issue at least

> This also happens if you use bevy_prototype_lyon and the heron crate. Tested with heron version 3 with the "debug-2d" feature on.
> 
> > thread 'main' panicked at 'Stage already exists: Shape.', C:\Users....cargo\registry\src\github.com-1ecc6299db9ec823\bevy_ecs-0.7.0\src\schedule\mod.rs:150:9
> 
> Cargo.toml line for heron: `heron = { version = "3", features = ["2d", "debug-2d"] }`
> 
> once you remove debug-2d this issue disappears.

**Comment by Nilirad** [comment]
Fixed by #163
